CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,72 @@
+- Add `ServiceTrafficDistribution` feature support in Antrea Proxy to enables Traffic Distribution for Services. ([#6604](https://github.com/antrea-io/antrea/pull/6604), [@hongliangl])
+- Add `antctl-darwin-arm64` to Antrea release assets. ([#6640](https://github.com/antrea-io/antrea/pull/6640), [@antoninbas])
+- Add documentation for `NodeLatencyMonitor` feature. ([#6561](https://github.com/antrea-io/antrea/pull/6561), [@antoninbas])
+
+### Changed
+
+- Uniform BGP router ID selection for IPv4 and IPv6. ([#6605](https://github.com/antrea-io/antrea/pull/6605), [@Atish-iaf])
+- Use the default protocol / port when the destination is a Service in Antrea Traceflows. ([#6601](https://github.com/antrea-io/antrea/pull/6601), [@Atish-iaf])
+- Add validations for `NodeNetworkPolicy` to fail invalid configs. ([#6613](https://github.com/antrea-io/antrea/pull/6613), [@Atish-iaf])
+- More robust system Tier creation / update. ([#6696](https://github.com/antrea-io/antrea/pull/6696), [@antoninbas])
+- Handle `ExternalIPPool` range changes in Egress controller. ([#6685](https://github.com/antrea-io/antrea/pull/6685), [@antoninbas])
+- Close connection to IPFIX collector explicitly on Stop for `FlowAggregator`. ([#6635](https://github.com/antrea-io/antrea/pull/6635), [@antoninbas])
+- Support `--random-fully` for iptables SNAT / MASQUERADE rules. ([#6602](https://github.com/antrea-io/antrea/pull/6602), [@antoninbas])
+- Unify the checker image and make it configurable when running `antctl check cluster`. ([#6579](https://github.com/antrea-io/antrea/pull/6579), [@tnqn])
+- Update the `Finalizer` of `ResourceExport` to be domain-qualified string. ([#6742](https://github.com/antrea-io/antrea/pull/6742), [@Dyanngg])
+- Avoid error log when unmarshalling config for Antrea Multi-cluster controller. ([#6744](https://github.com/antrea-io/antrea/pull/6744), [@antoninbas])
+- Upgrade Ubuntu to 24.04 (Noble). ([#6575](https://github.com/antrea-io/antrea/pull/6575), [@antoninbas])
+- Upgrade Go to 1.23. ([#6647](https://github.com/antrea-io/antrea/pull/6647), [@antoninbas])
+- Upgrade Suricata to 7.0. ([#6589](https://github.com/antrea-io/antrea/pull/6589), [@antoninbas])
+
+### Fixed
+
+- Match `dstIP` in `ClassifierTable` to fix a potential source MAC and IP mismatched issue on Windows when the `promiscuous` mode is enabled. ([#6528](https://github.com/antrea-io/antrea/pull/6528), [@XinShuYang])
+- Fix the checker image tag when running `antctl check cluster` with a released `antctl` binary. ([#6565](https://github.com/antrea-io/antrea/pull/6565), [@tnqn])
+- Use the same MTU as uplink for bridge port to fix a potential MTU mismatch issue when the traffic mode is changed. ([#6577](https://github.com/antrea-io/antrea/pull/6577), [@antoninbas])
+- Cache TTLs for individual IP addresses in DNS responses to avoid evicting valid IPs before they are expired. ([#6732](https://github.com/antrea-io/antrea/pull/6732), [@hkiiita])
```

Comment:
@antoninbas - my bad to delete your comment here by mistake. I think you were asking prioritizing this fix inc. reviews to get it in 2.2. Do not know how to restore the original comment.

Reply:
yes I was asking @hkiiita to prioritize work on this patch, and @tnqn @Dyanngg to prioritize its review